instance/tests/integration/integration_instance.py

```diff
@@ -42,6 +43,9 @@
 
 # Tests #######################################################################
 
+@override_settings(
+    DEFAULT_LMS_PREVIEW_DOMAIN_PREFIX='integ-',
```

Comment:
@bdero I'm fine with this, but since @antoviaque is our "upstream" reviewer here, he'll have to confirm as well :)